[BE] 로그 엔티티를 포함한 모든 엔티티의 칼럼명 형식 통일

## 🤷 리펙토링할 기능

## 🔨 상세 작업 내용

- [x] To-do 1: 다음 오류를 해결한다.
```
org.springframework.jdbc.BadSqlGrammarException: PreparedStatementCallback; bad SQL grammar [INSERT INTO request_response_log (method, path, request_body, response_status, response_body, created_at, client_ip) VALUES (?, ?, ?, ?, ?, ?, ?)]
	at org.springframework.jdbc.support.SQLStateSQLExceptionTranslator.doTranslate(SQLStateSQLExceptionTranslator.java:112) ~[spring-jdbc-6.1.12.jar!/:6.1.12]
	at org.springframework.jdbc.support.AbstractFallbackSQLExceptionTranslator.translate(AbstractFallbackSQLExceptionTranslator.java:107) ~[spring-jdbc-6.1.12.jar!/:6.1.12]
	at org.springframework.jdbc.support.AbstractFallbackSQLExceptionTranslator.translate(AbstractFallbackSQLExceptionTranslator.java:116) ~[spring-jdbc-6.1.12.jar!/:6.1.12]
	at org.springframework.jdbc.core.JdbcTemplate.translateException(JdbcTemplate.java:1548) ~[spring-jdbc-6.1.12.jar!/:6.1.12]
	at org.springframework.jdbc.core.JdbcTemplate.execute(JdbcTemplate.java:677) ~[spring-jdbc-6.1.12.jar!/:6.1.12]
	at org.springframework.jdbc.core.JdbcTemplate.execute(JdbcTemplate.java:701) ~[spring-jdbc-6.1.12.jar!/:6.1.12]
	at org.springframework.jdbc.core.JdbcTemplate.batchUpdate(JdbcTemplate.java:1047) ~[spring-jdbc-6.1.12.jar!/:6.1.12]
	at org.springframework.jdbc.core.JdbcTemplate.batchUpdate(JdbcTemplate.java:1063) ~[spring-jdbc-6.1.12.jar!/:6.1.12]
	at org.springframework.jdbc.core.JdbcTemplate.batchUpdate(JdbcTemplate.java:1054) ~[spring-jdbc-6.1.12.jar!/:6.1.12]
	at org.example.backend.log.repository.LogRepository.saveAllLogs(LogRepository.java:21) ~[!/:0.0.1-SNAPSHOT]
	at org.example.backend.log.service.LogService.saveAll(LogService.java:15) ~[!/:0.0.1-SNAPSHOT]
	at org.example.backend.log.service.BulkLogManager.lambda$processLogs$0(BulkLogManager.java:91) ~[!/:0.0.1-SNAPSHOT]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572) ~[na:na]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642) ~[na:na]
	at java.base/java.lang.Thread.run(Thread.java:1583) ~[na:na]
Caused by: java.sql.BatchUpdateException: Unknown column 'request_body' in 'field list'
	at com.mysql.cj.jdbc.exceptions.SQLError.createBatchUpdateException(SQLError.java:223) ~[mysql-connector-j-8.3.0.jar!/:8.3.0]
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeBatchSerially(ClientPreparedStatement.java:813) ~[mysql-connector-j-8.3.0.jar!/:8.3.0]
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeBatchInternal(ClientPreparedStatement.java:416) ~[mysql-connector-j-8.3.0.jar!/:8.3.0]
	at com.mysql.cj.jdbc.StatementImpl.executeBatch(StatementImpl.java:802) ~[mysql-connector-j-8.3.0.jar!/:8.3.0]
	at com.zaxxer.hikari.pool.ProxyStatement.executeBatch(ProxyStatement.java:127) ~[HikariCP-5.1.0.jar!/:na]
	at com.zaxxer.hikari.pool.HikariProxyPreparedStatement.executeBatch(HikariProxyPreparedStatement.java) ~[HikariCP-5.1.0.jar!/:na]
	at org.springframework.jdbc.core.JdbcTemplate.lambda$getPreparedStatementCallback$6(JdbcTemplate.java:1609) ~[spring-jdbc-6.1.12.jar!/:6.1.12]
	at org.springframework.jdbc.core.JdbcTemplate.execute(JdbcTemplate.java:658) ~[spring-jdbc-6.1.12.jar!/:6.1.12]
	... 12 common frames omitted
Caused by: java.sql.SQLSyntaxErrorException: Unknown column 'request_body' in 'field list'
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:121) ~[mysql-connector-j-8.3.0.jar!/:8.3.0]
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122) ~[mysql-connector-j-8.3.0.jar!/:8.3.0]
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeInternal(ClientPreparedStatement.java:912) ~[mysql-connector-j-8.3.0.jar!/:8.3.0]
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdateInternal(ClientPreparedStatement.java:1054) ~[mysql-connector-j-8.3.0.jar!/:8.3.0]
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeBatchSerially(ClientPreparedStatement.java:792) ~[mysql-connector-j-8.3.0.jar!/:8.3.0]
	... 18 common frames omitted
```


## 📄 참고 사항


## ⏰ 예상 소요 기간

--- a/backend/src/main/java/org/example/backend/log/domain/RequestResponseLog.java
+++ b/backend/src/main/java/org/example/backend/log/domain/RequestResponseLog.java
@@ -16,12 +16,22 @@ public class RequestResponseLog {
     private Long id;
 
     private String method;
+
     private String path;
+
+    @Column(name = "request_body")
     private String requestBody;
+
+    @Column(name = "request_status")
     private String responseStatus;
-    @Column(length = 10000)
+
+    @Column(name = "response_body", length = 10000)
     private String responseBody;
+
+    @Column(name = "created_at")
     private LocalDateTime createdAt;
+
+    @Column(name = "client_ip")
     private String clientIp;
 
 }

--- a/backend/src/main/java/org/example/backend/users/domain/entity/Users.java
+++ b/backend/src/main/java/org/example/backend/users/domain/entity/Users.java
@@ -12,7 +12,6 @@ import lombok.AccessLevel;
 import lombok.Builder;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
-import org.example.backend.users.exception.admin.AdminException;
 import org.springframework.security.crypto.password.PasswordEncoder;
 
 @Entity
@@ -36,6 +35,7 @@ public class Users {
 
     private String email;
 
+    @Column(name = "phone", unique = true)
     private String phoneN;
 
     private String department;

--- a/backend/src/main/resources/db/migration/V20250226__init.sql
+++ b/backend/src/main/resources/db/migration/V20250226__init.sql
@@ -152,11 +152,11 @@ CREATE TABLE IF NOT EXISTS request_response_log
     id             BIGINT NOT NULL AUTO_INCREMENT,
     method         VARCHAR(255),
     path           VARCHAR(255),
-    requestBody    VARCHAR(255),
-    responseStatus VARCHAR(255),
-    responseBody   VARCHAR(10000),
-    createdAt      DATETIME,
-    clientIp       VARCHAR(255),
+    request_body    VARCHAR(255),
+    response_status VARCHAR(255),
+    response_body   VARCHAR(10000),
+    created_at      DATETIME,
+    client_ip       VARCHAR(255),
     PRIMARY KEY (id)
 ) ENGINE = InnoDB
   DEFAULT CHARSET = utf8mb4;

--- a/backend/src/main/resources/db/migration/V20250226__init.sql
+++ b/backend/src/main/resources/db/migration/V20250226__init.sql
@@ -22,7 +22,7 @@ CREATE TABLE IF NOT EXISTS users
     password   VARCHAR(255) NOT NULL,
     username   VARCHAR(255),
     email      VARCHAR(255),
-    phoneN     VARCHAR(255),
+    phone      VARCHAR(255),
     department VARCHAR(255),
     role       VARCHAR(50)  NOT NULL,
     PRIMARY KEY (users_id)
@@ -149,9 +149,9 @@ CREATE TABLE IF NOT EXISTS jwt_blacklist
 
 CREATE TABLE IF NOT EXISTS request_response_log
 (
-    id             BIGINT NOT NULL AUTO_INCREMENT,
-    method         VARCHAR(255),
-    path           VARCHAR(255),
+    id              BIGINT NOT NULL AUTO_INCREMENT,
+    method          VARCHAR(255),
+    path            VARCHAR(255),
     request_body    VARCHAR(255),
     response_status VARCHAR(255),
     response_body   VARCHAR(10000),
@@ -178,7 +178,7 @@ ALTER TABLE thesis
         WITH PARSER ngram;
 
 -- users 더미 데이터
-INSERT INTO users (users_id, login_id, password, username, email, phoneN, role)
+INSERT INTO users (users_id, login_id, password, username, email, phone, role)
 VALUES (1, 'admin', '$2a$10$FKHTTHcEkAZZGW9XqGtPfOx.apKljbCLvYESM05YbLWzDynnacLPO', '관리자', 'admin@example.com',
         '010-1234-5678', 'ROLE_ADMIN');
 

